src/hardware/sensors/config/zed2i.yaml

```diff
@@ -77,9 +77,9 @@
             set_gravity_as_origin: true # If 'true' align the positional tracking world to imu gravity measurement. Keep the yaw from the user initial pose.
             floor_alignment: false # Enable to automatically calculate camera/floor offset
             initial_base_pose: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0] # Initial position of the `base_frame` in the map -> [X, Y, Z, R, P, Y]
-            init_odom_with_first_valid_pose: true # Enable to initialize the odometry with the first valid pose
+            init_odom_with_first_valid_pose: false # Enable to initialize the odometry with the first valid pose
             path_pub_rate: 2.0 # [DYNAMIC] - Camera trajectory publishing frequency
-            path_max_count: -1 # use '-1' for unlimited path size
+            path_max_count: 10 # use '-1' for unlimited path size
```

Comment:
Why 10?

Reply:
because unlimited is a huge path message
and i felt like 10 was a nice number